code/components/citizen-server-impl/include/state/SyncTrees_RDR3.h

```diff
@@ -14,7 +14,22 @@
 
 namespace fx::sync
 {
-struct CVehicleCreationDataNode { bool Parse(SyncParseState& state) { return true; } };
+struct CVehicleCreationDataNode
+{ 
+	uint32_t m_model;
+	ePopType m_popType;
+
+	bool Parse(SyncParseState& state)
```

Comment:
According to #960, we should stick to generic serialize instead of Parse/Unparse. Same for other data nodes.